hw/ip/rbox/data/rbox.hjson

```diff
@@ -0,0 +1,576 @@
+	  name: "pwrb_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "9",
+	  name: "key0_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "10",
+	  name: "key1_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+        { bits: "11",
+	  name: "key2_out_1",
+	  desc: "0: disable ;1: enable ",
+	}
+      ]
+    }
+    { name: "pin_out_ctl",
+      desc: "configure the pin out directly to override the function",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
+      resval: "2",
+      fields: [
+        { bits: "0",
+	  name: "bat_disable",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "1",
+	  name: "ec_rst_l",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "2",
+	  name: "pwrb_out",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "3",
+	  name: "key0_out",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "4",
+	  name: "key1_out",
+	  desc: "0: disable override;1: enable override",
+	}
+        { bits: "5",
+	  name: "key2_out",
+	  desc: "0: disable override;1: enable override",
+	}
+      ]
+    }
+    { name: "pin_out_value",
+      desc: "confiure the pin out value directly to override the function",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      hwqe: "true",
+      resval: "0",
+      fields: [
+        { bits: "0",
+	  name: "bat_disable",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "1",
+	  name: "ec_rst_l",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "2",
+	  name: "pwrb_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "3",
+	  name: "key0_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "4",
+	  name: "key1_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+        { bits: "5",
+	  name: "key2_out",
+	  desc: "0: override to 1b0; 1: override to 1b1",
+	}
+      ]
+    }
+    { name: "pin_in_value",
+      desc: "For SW to read the rbox inputs like GPIO",
+      swaccess: "ro",
+      hwaccess: "hrw",
+      hwqe: "true",
+      resval: "0",
+      fields: [
+        { bits: "0",
+	  name: "ac_present",
+	  desc: "raw ac_present value; before the invert logic",
+	}
+        { bits: "1",
+	  name: "ec_rst_l",
+	  desc: "raw ac_present value; before the invert logic",
+	}
+        { bits: "2",
+	  name: "pwrb_in",
+	  desc: "raw ac_present value; before the invert logic",
+	}
+        { bits: "3",
+	  name: "key0_in",
+	  desc: "raw ac_present value; before the invert logic",
+	}
+        { bits: "4",
+	  name: "key1_in",
+	  desc: "raw ac_present value; before the invert logic",
+	}
+        { bits: "5",
+	  name: "key2_in",
+	  desc: "raw ac_present value; before the invert logic",
+	}
+      ]
+    }
+    { name: "key_intr_ctl",
+      desc: "Define the keys or inputs that can trigger the interrupt ",
+      swaccess: "rw",
+      hwaccess: "hrw",
```

Comment:
yeah same here, this fees like `hwaccess` should be `hro`?

Reply:
Yes. I agree. Will fix.